openapi/sms/v1/instructions.md

```diff
@@ -0,0 +1,244 @@
+    Partner->>Vendor: Outbound SMS Messages
+    loop Each Delivery Report
+      Vendor->>Partner: SMS Delivery Receipt
+      Partner->>Emarsys: Message Delivery Report
+      Emarsys->>Client: Message Reporting
+      Note left of Client: User views reports via UI
+    end
+  end
+```
+
+#### Outbound Messages
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  participant Vendor as SMS Vendor API
+  loop Each SMS campaign trigger
+    Client-->Emarsys: Triggers Campaign
+    Emarsys->>Partner: Outbound Messages
+    Partner->>Vendor: Outbound SMS Messages
+  end
+```
+
+- [Create Outbound Message Batch](../partner-service.yaml/paths/~1clients~1{clientId}~1messages/post)
+  - Request Payload
+    ```yaml json_schema
+    $ref: '../partner-service.yaml#/components/schemas/OutboundMessageBatch'
+    ```
+
+#### Delivery Reports
+
+Delivery reports are communicated via a callback to Emarsys SMS:
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  participant Vendor as SMS Vendor API
+  loop Each Delivery Report
+      Vendor->>Partner: SMS Delivery Receipt
+      Partner->>Emarsys: Message Delivery Report
+      Emarsys->>Client: Message Reporting
+      Note left of Client: User views reports via UI
+  end
+```
+
+---
+
+- [Create Delivery Report Batch](../partner-callbacks.yaml/paths/~1clients~1{clientId}~1deliveryReports)
+
+  ***
+
+  - ##### Request
+    ***
+    - Delivery Report:
+      ```yaml json_schema
+      $ref: '../partner-callbacks.yaml#/components/schemas/DeliveryReport'
+      ```
+      ***
+  - ##### Response
+    ***
+    - Delivery Report Error:
+      ```yaml json_schema
+      $ref: '../partner-callbacks.yaml#/components/schemas/DeliveryReportError'
+      ```
+      ***
+    - Invalid Format Error Batch Response Payload:
+      ```yaml json_schema
+      $ref: '../common.yaml#/components/schemas/InvalidFormatErrorBatch'
+      ```
+      ***
+
+---
+
+### Inbound Messages
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  participant Vendor as SMS Vendor API
+  loop Each inbound message
+    Vendor->>Partner: Mobile Originated SMS Message
+    Partner->>Emarsys: Inbound Message
+    Emarsys->>Client: Inbound Messages
+    Note left of Client: Messages used in automations
+  end
+```
+
+- [Create Inbound Messages Batch](../partner-callbacks.yaml/paths/~1clients~1{clientId}~1inboundMessages)
+  - Inbound Message Batch Request Payload:
+    ```yaml json_schema
+    $ref: '../partner-callbacks.yaml#/components/schemas/InboundMessageBatch'
+    ```
+  - Invalid Format Error Batch Response Payload:
+    ```yaml json_schema
+    $ref: '../common.yaml#/components/schemas/InvalidFormatErrorBatch'
```

Comment:
```suggestion
    $ref: './common.yaml#/components/schemas/InvalidFormatErrorBatch'
```

Reply:
we navigate to the parent directory here because in stoplight markdown files are places in a `docs` directory (within the root) whereas the OpenAPI yaml files are directly in the root